piker/data/feed.py

```diff
@@ -188,6 +191,22 @@ async def _setup_persistent_brokerd(
         await trio.sleep_forever()
 
 
+async def start_backfill(
```

Comment:
Ok so there is a bunch of new code for this in #302 that hasn't yet been factored out. I wasn't sure if that should all come as part of a "backend ohlc history support" PR (aka more of less what #305 is) or be included here?

Technically this PR will land with minimal backend support (i think just `ib`?) so it's more about how much code ya'll want test review :joy:

Reply:
Yeah this kinda stuff was the main reason for dropping this PR and just aggregating it all in to #308.